seqr/utils/search/utils.py

```diff
@@ -404,7 +410,8 @@ def _parse_inheritance(search, samples, previous_search_results):
                     f'Unable to search for comp-het pairs with dataset type "{invalid_type}". This may be because inheritance based search is disabled in families with no loaded affected individuals'
                 )
 
-    return samples
+    if not has_location_filter:
```

Comment:
is the technical reason for requiring a location filter for multiple project/family searches because the search takes too long without it?